cabal.project

```diff
@@ -36,6 +36,18 @@ package hashes
       /opt/local/lib
       /usr/local/opt/openssl/lib/
 
+package rocksdb-haskell
+    extra-include-dirs:
+      /opt/local/include
+      /usr/local/opt/gflags/include
+      /usr/local/opt/rocksdb/include
+      /usr/local/include
+    extra-lib-dirs:
+      /opt/local/lib
+      /usr/local/opt/gflags/lib
+      /usr/local/opt/rocksdb/lib
+      /usr/local/lib
+
```

Comment:
shouldn't rocksdb be installed within the scope of the `rocksdb-haskell` package within the `~/.cabal` directory?